index.ts

```diff
@@ -0,0 +1,104 @@
+import { launch } from './run.js'
+import {
+  BatchWriteItemCommand,
+  DynamoDBClient,
+  UpdateTableCommand,
+} from '@aws-sdk/client-dynamodb'
+import _arcFunctions from '@architect/functions'
+import { marshall } from '@aws-sdk/util-dynamodb'
+import { access, constants, readFile } from 'node:fs/promises'
+import chunk from 'lodash/chunk.js'
+import dedent from 'ts-dedent'
+
+let local: Awaited<ReturnType<typeof launch>>
+
+export const deploy = {
+  async services() {
+    if (process.env.ARC_DB_EXTERNAL) local = await launch()
+  },
+}
+
+export const sandbox = {
+  // @ts-expect-error: The Architect plugins API has no type definitions.
+  async start({ inventory: { inv }, arc }) {
+    const dynamodbClient = new DynamoDBClient({
+      region: inv.aws.region,
+      endpoint: `http://localhost:${process.env.ARC_TABLES_PORT}`,
+      requestHandler: {
+        requestTimeout: 10_000,
+        httpsAgent: { maxSockets: 500 }, // Increased from default to allow for higher throughput
+      },
+    })
+    const seedFile = arc['architect-plugin-dynamodb-local'].find(
+      (item: string[]) => item[0] == 'seedFile'
+    )[1]
+    const client = await _arcFunctions.tables()
+    if (seedFile) {
+      if (['sandbox-seed.json', 'sandbox-seed.js'].includes(seedFile)) {
+        console.log(dedent`
```

Comment:
Should this be a warning?

```suggestion
        console.warn(dedent`
```

Reply:
I don't think it makes a difference here. I tested both and they output identically